addressfinder.gemspec

```
@@ -11,13 +11,14 @@ Gem::Specification.new do |gem|
   gem.summary       = 'Provides easy access to AddressFinder APIs'
   gem.homepage      = 'https://github.com/AddressFinder/addressfinder-ruby'
 
-  gem.files         = `git ls-files`.split($/)
+  gem.files         = `git ls-files`.split($/).select{|f| f.match(%r{^lib|.gemspec|.md})}
```

Haven't tested the regex but this could tighten it:
1. Make `.` match the actual dot, not any char
2. Make sure you are matching the extension not a substring anywhere in the filename

```suggestion
  gem.files         = `git ls-files`.split($/).select{|f| f.match(%r{^lib|\.gemspec$|\.md$})}
```

---

This is all under our own control but won't hurt to tighten a bit, _in case_